core-bundle/src/Resources/config/routing.yml

```diff
@@ -10,3 +10,7 @@ contao_backend_redirect:
         _controller: Symfony\Bundle\FrameworkBundle\Controller\RedirectController::redirectAction
         route: contao_backend
         permanent: true
+
+contao_images:
```

Comment:
We should move this above `contao_backend_redirect`.

Reply:
98384c9ce46b4455ee67c1e260db569b82f0883c